decidim-admin/app/helpers/decidim/admin/newsletters_helper.rb

```diff
@@ -65,7 +65,7 @@ def sent_to_users(newsletter)
 
       def sent_to_spaces(newsletter)
         html = "<p style='margin-bottom:0;'> "
-        newsletter.sended_to_partipatory_spaces.try(:each) do |type|
+        newsletter.sended_to_participatory_spaces.try(:each) do |type|
```

Comment:
if we are changing this, why not fixing properly?  
```suggestion
        newsletter.sent_to_participatory_spaces.try(:each) do |type|
```